lib/msal-node/src/cache/TokenCache.ts

```diff
@@ -35,14 +35,16 @@ export class TokenCache {
     private hasChanged: boolean;
     private cacheSnapshot: string;
     private readonly persistence: ICachePlugin;
+    private logger: Logger;
 
-    constructor(storage: Storage, cachePlugin?: ICachePlugin) {
+    constructor(storage: Storage, cachePlugin?: ICachePlugin, logger?: Logger) {
```

Comment:
I would make `logger` required, since all of the code in this class assumes it is provided.